### PR TITLE
SBT project changes

### PR DESCRIPTION
**Pull request by alfonsorr**

### CODE REVIEW

**Review by zeotuan — COMMENTED** [review]
Note: `randGamma` test failed for spark 3.4 and 3.5 but that should be fine as it will be fixed 
https://github.com/mrpowers-io/spark-daria/pull/165

LGTM!

**Review by zeotuan — APPROVED** [review]

**Comment by alfonsorr** [comment]
I see the reason why you didn't want to merge them splitter, I will drop the latest spark versions, and you add them

File: .github/workflows/unsafe-ci.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        spark: ["3.2.4", "3.3.4"]
+        spark: ["3.0.3", "3.1.3", "3.2.4", "3.3.4"]
```
Comment (zeotuan) [inline .github/workflows/unsafe-ci.yml:14]: @alfonsorr RandGamma will probably not work with 3.0 and 3.1 too which I am thinking of adding after support for 3.4 is checked in

Comment (alfonsorr) [reply .github/workflows/unsafe-ci.yml:14]:
```suggestion
        spark: ["3.2.4", "3.3.4"]
```

**Pull request merged**